src/ServiceControl.Audit/Auditing/MessagesView/GetMessagesController.cs

```diff
@@ -6,10 +6,17 @@ namespace ServiceControl.Audit.Auditing.MessagesView
 
     public class GetMessagesController : ApiController
```

Comment:
Should we mention in a comment that messages routes need to be on this controller to not mess up catch all?

Reply:
Added comment